### PR TITLE
WIP: Add the swift serverspec

### PR DESCRIPTION
**Pull request by vlope093**
Dependent on PR #1554

### CODE REVIEW

File: roles/swift-common/templates/etc/serverspec/swift_serverspec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  end     
+end
+
+describe file('/var/log/syslog') do 
+    it { should be_owned_by 'syslog' }
+    it { should be_grouped_into 'adm' }
+    it { should be_mode 640 }   
+end
+
+describe file('/dev/log') do
```
Comment (j2sol) [inline roles/swift-common/templates/etc/serverspec/swift_serverspec.rb]: This and the syslog file above seem pretty generic, and not related to swift. Why are they here?

**Comment by j2sol** [comment]
I suspect this will need to be broken up across the various swift roles, since not every system will run the same processes and log to the same files.

**Comment by j2sol** [comment]
The /var/log and /var/log/syslog checks should be moved to OS server specs, not here in swift.

**Comment by j2sol** [comment]
ok to test

**Comment by j2sol** [comment]
Looks like a file is missing `roles/swift-common/tasks/severspec.yml`

**Comment by pwplusnick** [comment]
@j2sol: `roles/swift-common/tasks/serverspec.yml` is definitely there. I'm not sure why Jenkins threw that error, though. Strange. Any ideas?

File: roles/swift-common/tasks/main.yml [hunk 2]
```diff
@@ -97,3 +97,8 @@
 - name: disable swift directory parsing by updatedb
   command: updatedb -e /srv/node
 
+- include: severspec.yml
```
Comment (j2sol) [inline roles/swift-common/tasks/main.yml]: s/severspec/serverspec/

Small typo, big impact.

**Pull request merged**